src/components/Crypto/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {CryptoTypes} from '../../types';
+import {
+  CryptoContainer,
+  NameContainer,
+  Price,
+  Logo,
+  Increase,
+  Decrease,
+  Name,
+  Slug,
+  Arrow,
+  Arrowdown,
+  Arrowup,
+  Percentage,
+} from '../Crypto/styles';
+
+const Crypto = ({name, slug, price, market_porcentage, image}: CryptoTypes) => {
```

Comment:
you can send the object directly, and change the type for a prop type:


import React, {FC} from 'react'

interface Props {
  crypto: CryptoTypes;
}

const Crypto: FC<Props> = (item: {name, slug, price, market_porcentage, image}) => {